examples/knative/python/kn-py-vm-attr/handler.py

```diff
@@ -4,85 +4,40 @@
 from vcenter import Session
 from datetime import date
 
-logging.basicConfig(level=logging.DEBUG,format='%(asctime)s %(levelname)s %(name)s %(threadName)s : %(message)s')
+logging.basicConfig(
+    level=logging.DEBUG,
+    format='%(asctime)s %(levelname)s %(name)s %(threadName)s : %(message)s'
+)
 
 app = Flask(__name__)
-@app.route('/', methods=['GET','POST'])
+@app.route('/', methods=['POST'])
 def echo():
-    if request.method == 'GET':
-        sc = 200
-        msg = 'POST to this endpoint to apply custom attributes to a VM object within a cloudEvent message'
-        message = {
-            'status': sc,
-            'message': msg,
-        }
-        resp = jsonify(message)
-        resp.status_code = sc
-        return resp
-
-    if request.method == 'POST':
-        try:
-            event = from_http(request.headers, request.get_data(),None)
+    try:
+        event = from_http(request.headers, request.get_data(),None)
 
-            data = event.data
-            # hack to handle non JSON payload, e.g. xml
-            if not isinstance(data,dict):
-                data = str(event.data)
-
-            cevent = {
-                "attributes": event._attributes,
-                "data": data
+        if event._attributes.get("datacontenttype").lower() != "application/json":
```

Comment:
nice, please add this to the README so that only JSON-encoded payloads are supported